carbon-design-system/carbon-addons-iot-react
# [DateTimePicker]: Added validation for start/end time

Closes #3134
**Summary**

- When setting a custom time range for DateTimePicker one can select a start time that's later than the end time.

**Change List (commits, features, bugs, etc)**

- Added validation to prevent a start time after the end time to be selected. An error message is displayed.

**Acceptance Test (how to verify the PR)**

- Try to select a start time after the end time.
- Try to select an end time before the start time.

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Run unit tests 



**Things to look for during review**
